### PR TITLE
Enable the email alert

### PR DESCRIPTION
**Pull request by athiruma**
## Type of change
Note: Fill **x** in []
- [x] bug
- [ ] enhancement
- [x] documentation
- [ ] dependencies

## Description
Enabled the email alert.

### CODE REVIEW

File: cloud_governance/policy/aws/ec2_idle.py [hunk 1]
```diff
@@ -143,8 +143,11 @@ def __trigger_mail(self, tags: list, resource_id: str, days: int, ec2_type: str
             special_user_mails = self._literal_eval(self._special_user_mails)
             user, instance_name = self._get_tag_name_from_tags(tags=tags, tag_name='User'), self._get_tag_name_from_tags(tags=tags, tag_name='Name')
             to = user if user not in special_user_mails else special_user_mails[user]
+            email_id = self._get_tag_name_from_tags(tags=tags, tag_name='Email')
             ldap_data = self._ldap.get_user_details(user_name=to)
             cc = [self._account_admin, f'{ldap_data.get("managerId")}@redhat.com']
+            if email_id:
```
Comment (ebattat) [inline cloud_governance/policy/aws/ec2_idle.py:149]: What the purpose of this  email adding ?

File: jenkins/tenant/aws/common/run_policies.py [hunk 2]
```diff
@@ -27,7 +27,7 @@
 os.system(f"echo Polices list: {policies_not_in_action}")
 for region in regions:
     for policy in policies_not_in_action:
-        os.system(f"""podman run --rm --name cloud-governance-poc-haim --net="host" -e MANAGER_EMAIL_ALERT="False" -e EMAIL_ALERT="False" -e account="{account_name}" -e policy="{policy}" -e AWS_ACCESS_KEY_ID="{access_key}" -e AWS_SECRET_ACCESS_KEY="{secret_key}" -e AWS_DEFAULT_REGION="{region}" -e dry_run="yes" -e LDAP_HOST_NAME="{LDAP_HOST_NAME}" -e es_host="{ES_HOST}" -e es_port="{ES_PORT}" -e policy_output="s3://{s3_bucket}/{LOGS}/{region}" -e log_level="INFO" quay.io/ebattat/cloud-governance:latest""")
+        os.system(f"""podman run --rm --name cloud-governance-poc-haim --net="host" -e MANAGER_EMAIL_ALERT="True" -e EMAIL_ALERT="True" -e account="{account_name}" -e policy="{policy}" -e AWS_ACCESS_KEY_ID="{access_key}" -e AWS_SECRET_ACCESS_KEY="{secret_key}" -e AWS_DEFAULT_REGION="{region}" -e dry_run="yes" -e LDAP_HOST_NAME="{LDAP_HOST_NAME}" -e es_host="{ES_HOST}" -e es_port="{ES_PORT}" -e policy_output="s3://{s3_bucket}/{LOGS}/{region}" -e log_level="INFO" quay.io/ebattat/cloud-governance:latest""")
```
Comment (ebattat) [inline jenkins/tenant/aws/common/run_policies.py:30]: Do you enable it for all not for Haim ?

File: cloud_governance/policy/aws/ec2_idle.py (hunk 1 above)
Comment (athiruma) [reply cloud_governance/policy/aws/ec2_idle.py:149]: if user name is not in LDAP, we need to check Email tag.

File: jenkins/tenant/aws/common/run_policies.py (hunk 2 above)
Comment (athiruma) [reply jenkins/tenant/aws/common/run_policies.py:30]: It is commonfolder, so it should be enabled by default

**Pull request closed without merging**